Conventional/KnownPaths.cs

```diff
@@ -0,0 +1,12 @@
+using System.IO;
+
+namespace Conventional
+{
+    public static class KnownPaths
+    {
+        public static string SolutionRoot
+        {
+            get { return Path.GetFullPath(Path.Combine(Directory.GetCurrentDirectory(), @"../../../")); }

```

Comment:
Will this work with the sln root a level above the projects? ie if I dump my projects in /src but keep the sln at the root?